tests/examples/examples_ft.cpp

```diff
@@ -215,16 +201,15 @@ TEST_F(examples_ft, transformation)
 {
     const std::string transformation_dir = TEST_TEMP_DIR;
 #ifdef _WIN32
-    const std::string delim = "\\";
-    const std::string transformation_path = "bin\\transformation_example.exe";
+    const std::string transformation_path = "bin/transformation_example.exe";
```

Comment:
Since this is a call to a shell command, I think you can omit the .exe on windows as well, and eliminate another #if